expo/scheduler/views.py

```diff
@@ -906,6 +782,143 @@ def add_event(request, eventitem_id, event_type='Class'):
                                       'event_type': event_type})
 
 
+def edit_event(request, scheduler_event_id, event_type='class'):
+    '''
+    Add an item to the conference schedule and/or set its schedule details
+    (start time, location, duration, or allocations)
+    Takes a scheduler.Event id
+    '''
+    profile = validate_perms(request, ('Scheduling Mavens',))
+
+    try:
+        item = Event.objects.get_subclass(id=scheduler_event_id)
+    except:
+        raise Exception("Error code XYZZY: Couldn't get an item for id")
+        # TO DO: log this exception
+    if request.method == 'POST':
+        event_form = EventScheduleForm(request.POST,
+                                       instance=item,
+                                       prefix='event',
+                                       conference=item.get_conference())
+        if event_form.is_valid():
```

Comment:
== BLOCKER == 
Problem - I can't edit a scheduled event.
Steps to reproduce:
1 - go to: http://localhost:8282/scheduler/list/GenericEvent
2 - create a new event - http://localhost:8282/create_event/GenericEvent
3 - select the event and add a session
4 - save it (looks good on event list)
5 - edit it
6 - change something (I changed Duration) - save it.
Error encountered:  

AttributeError at /scheduler/edit/GenericEvent/1418

'Event' object has no attribute 'get_conference'